### ISSUE TITLE
Populate User Guide page with real content

### ISSUE THREAD

**Issue by hazelshapiro** [nsidc/usaon-benefit-tool#231]
What should go here - test some edits using markdown

https://github.com/nsidc/usaon-benefit-tool/blob/main/usaon_benefit_tool/templates/user_guide.md

**Comment by mfisher87** [comment]
@hazelshapiro I believe this is a task you intend to complete yourself, so I assigned it to you! If that's not correct, please remove assignment and let me know :)

**Comment by hazelshapiro** [comment]
Yes, I'll take this on. @mfisher87 for your reference - here are the user guides for the previous version of the tool (using spreadsheets, etc)
Scoping guidance: https://docs.google.com/document/d/1gQGftJuOSLWLYXh5onBtF9PRoBlFbrQtotS6XuGItrE/edit
Rating guidance: https://docs.google.com/document/d/1-9zxLILJjSFc9gTVkbOUYyfZWIzAFdRFNYxo36HFlBY/edit
Apologies, I know the google docs are not a great practice.

**Comment by mfisher87** [comment]
No sweat! If the info in the docs isn't private, we should be able to make them more easily viewable by selecting `File > Share > Publish to web...`

**Comment by rmarow** [comment]
We have done this. https://github.com/nsidc/usaon-benefit-tool/issues/362 this issue covers much of what was done!
